daemon/daemon_unix.go

```diff
@@ -556,8 +556,8 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 	return warnings, nil
 }
 
-// platformReload updates configuration with platform specific options
-func (daemon *Daemon) platformReload(config *Config) map[string]string {
+// reloadPlatform updates configuration with platform specific options
+func (daemon *Daemon) reloadPlatform(config *Config, attributes map[string]string) {
```

Comment:
Please add a comment that callee updates the `attributes` ?

Generally I don't prefer args to be written.
An alternative design I can come up with is to keep returning `map`, and compose `map`s to a single `map` with some utility function.

i.e.
```go
mb := mapBuilder()
mb.Append(daemon.reloadPlatform(config))
mb.Append(daemon.reloadBlahBlahBlah(config))
attributes := mb.Map()
```

But maybe too complicated.